### PR TITLE
[WIP]ShvetsovAM_lab1

### PR DESCRIPTION
**Pull request by Ksallle**

### CODE REVIEW

File: ShvetsovAM/lab1/Calculator/Calculator/Calculator.cpp [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+		break;
+	case '*': result = first*second;
+		break;
+	case '/': result = first / second;
+
+		if (second == 0)
+		{
+			cout << "������: �� ���� ������ ������";
+			_getch();
+			return 0;
```
Comment (Toxa-man) [inline ShvetsovAM/lab1/Calculator/Calculator/Calculator.cpp]: Do not use return statement here
Normally you don't wont calculator you use to shut down every time you make a mistake

File: ShvetsovAM/lab1/Calculator/Calculator/Calculator.cpp [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+		}
+		break;
+	default: cout << "������: ����������� ����";
+		_getch();
+		return 0;
```
Comment (Toxa-man) [inline ShvetsovAM/lab1/Calculator/Calculator/Calculator.cpp]: The same as above

**Comment by Ksallle** [comment]
You say, that i cant use "return 0' more than 1 time. But without this my program dont work

**Comment by Ksallle** [comment]
Last files via is correct. Check this

File: ShvetsovAM/lab1/Calculator.code [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+	char sign;
+	cout << "Введите первое число ";
+	cin >> first;
+	cout << "Введите второе число ";
+	cin >> second;
+	cout << "Введите знак ";
+	cin >> sign;
+	switch (sign)
+	{
+	case '+': cout << first << " " << sign << " " << second << " = " << first + second;
```
Comment (Toxa-man) [inline ShvetsovAM/lab1/Calculator.code]: You have to store the result in variable and print it after all checks

Comment (Ksallle) [reply ShvetsovAM/lab1/Calculator.code]: I dont know how to do this
It's not working
But how i did - working

**Pull request closed without merging**